src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/VedtaksperioderMedBegrunnelser/VedtaksperioderMedBegrunnelser.tsx

```diff
@@ -74,6 +74,10 @@ const VedtaksperioderMedBegrunnelser: React.FC<IVedtakBegrunnelserTabell> = ({
                 }
                 åpenBehandling={åpenBehandling}
             />
+            <TrekkILøpendeUtbetalingListe
+                visTrekkILøpendeUtbetalinger={true}
+                åpenBehandling={åpenBehandling}
+            />
```

Comment:
Ville lagt denne komponentent i `OppsummeringVedtak.tsx` og ikke inne i `VedtaksperioderMedBegrunnelse` for å vise tydelig i koden at dette er to ulike "blokker" som ikke hører sammen.

Reply:
Her bør det også lages en variabel som styrer `visTrekkILøpendeUtbetalinger={X}`

Reply:
Einig!

Reply:
Hele denne må bak feature toggle også tror jeg